apps/seg/itkimage2segimage.cxx

```diff
@@ -43,12 +45,15 @@ int main(int argc, char *argv[])
   if(dicomDirectory.size()){
     OFList<OFString> fileList;
     cout << "Searching recursively " << dicomDirectory << " for DICOM files" << endl;
+#if _WIN32
+    replace(dicomDirectory.begin(), dicomDirectory.end(), '/', '\\');
```

Comment:
need to confirm there is no OS-independent definition of file separator in dcmtk